nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/work/StreamWorkManager.java

```diff
@@ -212,9 +190,9 @@ public int getApplicationStartedOrder() {
     @Override
     public void start(ComponentContext context) {
         init();
-        storeState = Boolean.parseBoolean(Framework.getService(ConfigurationService.class).getProperty(STORESTATE_KEY));
-        stateTTL = Long.parseLong(
-                Framework.getService(ConfigurationService.class).getProperty(STATETTL_KEY, STATETTL_DEFAULT_VALUE));
+        ConfigurationService configuration = Framework.getService(ConfigurationService.class);
+        storeState = Boolean.parseBoolean(configuration.getProperty(STORESTATE_KEY));
```

Comment:
You can use `storeState = configuration.isBooleanPropertyTrue(STORESTATE_KEY)` here